components/cd-image-grid/cd-image-grid.css

```diff
@@ -1,6 +1,11 @@
+:root {
+  --cd-grid--item-size: 4rem;
```

Comment:
I think this should be named `--cd-grid-image--item-size` to avoid confusion (and potential collision in the future) with the `cd-grid` component.